_data/toc/installation-guide.yml

```diff
@@ -258,6 +258,10 @@ pages:
             - label: Update the Magento database schema and data
               url: /install-gde/install/cli/install-cli-subcommands-db-upgr.html
 
+            - label: Configure the lock provider
+              url: /install-gde/install/cli/install-cli-subcommands-lock.html
+              include_versions: ["2.1","2.2","2.3"]
```

Comment:
I don't see a 2.1 file. Should this be limited to 2.2 and 2.3?

Reply:
Only supported on 2.2.5 and later. Thx for catching that.